Area51/Sources/ContributionsController.swift

```diff
@@ -0,0 +1,72 @@
+import UIKit
+
+class ContributionController: UITableViewController {
+
+    var listOfContributors = [Contributor]()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.register(UINib(nibName: "ContributorCell", bundle: nil), forCellReuseIdentifier: "ContributorCell")
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        DispatchQueue.global(qos: .background).async {
+            self.getContributors { (res) in
+                switch res {
+                case .success(let contributors):
+                    contributors.forEach({ (contributor) in
+                        self.listOfContributors.append(contributor)
+                    })
+                case .failure(let err):
+                    print(err)
+                }
+            }
+        }
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        DispatchQueue.main.async {
```

Comment:
- call super.viewDidAppear
- No need for dispatching to main, this is already executed on main.